packages/react-components/react-nav-preview/library/src/components/sharedNavStyles.styles.ts

```diff
@@ -57,8 +57,12 @@ export const useRootDefaultClassName = makeResetStyles({
     backgroundColor: navItemTokens.backgroundColorHover,
   },
   ':active': {
+    zIndex: 1,
```

Comment:
I always hesitate to bake in z indexes to shared components because product code often has their own z indexing strategy that can clash with ours. I suppose we're in the clear if it's easy to override ours.